src/components/annotations/index.js

```diff
@@ -6,25 +6,24 @@
 * LICENSE file in the root directory of this source tree.
 */
 
-
 'use strict';
 
 var drawModule = require('./draw');
 var clickModule = require('./click');
 
 module.exports = {
-    moduleType: 'component',
-    name: 'annotations',
+  moduleType: 'component',
+  name: 'annotations',
 
```

Comment:
Interesting, here `prettier` does respect vertical formatting and keeps the empty line.